cloudinary_cli/modules/regen_derived.py

```diff
@@ -25,7 +24,7 @@
 @option("-A", "--auto_paginate", is_flag=True, default=False,
         help="Auto-paginate Admin API calls.")
 @option("-F", "--force", is_flag=True,
-        help="Skip initial and auto-paginate confirmation.")
+        help="Skip initial confirmation.")
```

Comment:
Maybe change to `Skip confirmation.` or `Skip user confirmation.`. like, `initial` is kind of ambiguous, no context.
@carlevison, WDYT?

Reply:
@const-cloudinary 'Skip confirmation.' is fine.

Reply:
updated to `Skip confirmation.`